Cargo.toml

```diff
@@ -20,14 +20,14 @@ tokio-core = "0.1.12"
 tokio-process = "0.1.5"
 tokio-timer = "0.2"
 
-clap = "2.20.0"
-semver = "0.11"
-blake2 = "0.9"
+clap = { version = "3.1", features = ["derive"] }
+semver = "1.0"
+blake2 = "0.10"
 typed-arena = "2"
-dirs = "3"
+dirs = "4"
 pathdiff = "0.2"
 common-path = "1.0.0"
-itertools = "0.9"
+itertools = "0.10"
```

Comment:
[According to deps.rs](https://deps.rs/repo/github/pulp-platform/bender), the dependencies `futures` and `tokio-process` are also outdated. Is it possible to update them as well, or is this out of scope and should be tracked in a new issue?

Reply:
I started looking into this but the breaking changes in [`tokio`](https://docs.rs/tokio/latest/tokio/) and `futures` are significant and would require rewriting large parts of bender, and I don't currently have the time to dig into this.

Reply:
Okay, makes sense. Shall we open an issue to track this?

Reply:
Done: #89